de/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: news_post
+title: "Ruby 2.3.0-preview1 veröffentlicht"
+author: "naruse"
+translator: "Marvin Gülker"
+date: 2015-11-11 14:00:00 +0000
+lang: de
+---
+
+Wir freuen uns, die Veröffentlichung von Ruby 2.3.0-preview1
+bekanntgeben zu können.
+
+Ruby 2.3.0-preview1 ist die erste Vorschau auf Ruby 2.3.0, in die
```

Comment:
Nein, Geschmackssache. Mein Gedanke war, dass die Verbesserungen ja nicht nur in die Vorschau einfließen, sondern auch in die endgültige Version 2.3 - aber kann gerne so bleiben, Lesefluss ist so sicherlich besser.